contracts/src/periphery/cert/EigenDACertVerifier.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IVersionedEigenDACertVerifier} from "src/periphery/cert/interfaces/IVersionedEigenDACertVerifier.sol";
+import {IEigenDACertVerifierBase} from "src/periphery/cert/interfaces/IEigenDACertVerifierBase.sol";
+
+import {IEigenDAThresholdRegistry} from "src/core/interfaces/IEigenDAThresholdRegistry.sol";
+import {IEigenDASignatureVerifier} from "src/core/interfaces/IEigenDASignatureVerifier.sol";
+
+import {EigenDATypesV1 as DATypesV1} from "src/core/libraries/v1/EigenDATypesV1.sol";
+import {EigenDATypesV2 as DATypesV2} from "src/core/libraries/v2/EigenDATypesV2.sol";
+
+import {EigenDACertVerificationLib as CertLib} from "src/periphery/cert/libraries/EigenDACertVerificationLib.sol";
+
+contract EigenDACertVerifier is IEigenDACertVerifierBase, IVersionedEigenDACertVerifier {
```

Comment:
Based on this diagram, there would be some kind of `IEigenDACertVerifier` interface. Did you decide that wasn't necessary?

<img width="680" alt="image" src="https://github.com/user-attachments/assets/068431de-e2bc-47ff-b8c9-b9bea30cad56" />

Reply:
I decided it was not necessary because the contract itself can be the interface. Then there is no chance of misalignment between the interface and the cert verifier functions.

Reply:
I thought the point of the interface was so that we only have to keep the old interfaces around in the repo for proxy bindings, and not all the contracts. 

But agree we can/should aim to merge soon. We can add the interfaces once we get a v4